integration/spark/src/main/scala/org/apache/spark/sql/execution/command/management/CommonLoadUtils.scala

```diff
@@ -568,8 +567,7 @@ object CommonLoadUtils {
         .getColumnSchemaList.size()))
         .asInstanceOf[Array[ColumnSchema]]
     } else {
-      colSchema = colSchema.filterNot(x => x.isInvisible || x.isComplexColumn ||
-                                           x.getSchemaOrdinal == -1)
+      colSchema = colSchema.filterNot(x => x.isInvisible || x.isComplexColumn)
```

Comment:
why remove getSchemaOrdinal check? we do not judge spatial column by schema ordinal as -1. And x.getSchemaOrdinal was not introduced with spatial support PR. Same applies to if case above.